doc/admin-api.md

```diff
@@ -662,10 +662,12 @@ Return response from etcd currently.
 |key|True|Private key|https private key||
 |certs|False|An array of certificate|when you need to configure multiple certificate for the same domain, you can pass extra https certificates (excluding the one given as cert) in this field||
 |keys|False|An array of private key|https private keys. The keys should be paired with certs above||
-|sni|True|Match Rules|https SNI||
+|sni|True, can't be used with `snis`|Match Rules|https SNI||
```

Comment:
I think we can remove the field `sni` from doc, we'll only support `snis` in the future.

Reply:
@membphis 
Updated.